server/api/reports.py

```diff
@@ -891,17 +923,22 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
         is_audited = batch.id in audit_results_by_batch
         row += [pretty_boolean(is_audited)]
 
+        is_combined = batch.id in combined_sub_batch_ids
+
         for contest in contests:
-            reported_results = (
-                batch.jurisdiction.batch_tallies[batch.name][contest.id]
-                if contest.id in batch.jurisdiction.batch_tallies[batch.name]
-                else None
+            reported_results = batch.jurisdiction.batch_tallies[batch.name].get(
+                contest.id
             )
             if reported_results is not None:
-                for choice in contest.choices:
-                    total_reported_results[contest.id][choice.id] += reported_results[
-                        choice.id
-                    ]
+                # For combined batches, we need to add the combined batch reported
+                # results to the total, since it may include some unsampled batches.
+                # However, for audit results, we want to use only the sampled
+                # batches to avoid double counting.
```

Comment:
Hmm, I'm having some trouble following what the double counting here would be

Reply:
Yeah, I got confused reading this comment again lol. Edited to hopefully clarify 3c69ed2f

Reply:
After rebase, that's actually: 153f72029c11dc6fa562f6bd1e86b227a8009eec